roles/openshift_project_request_template/tasks/main.yml

```diff
@@ -2,7 +2,7 @@
 - name: Create temp file for template
   command: mktemp /tmp/openshift-ansible-XXXXXX.yaml
   register: mktemp
-  changed_when: False
+  changed_when: False | bool
```

Comment:
This change is probably not needed

Reply:
If there are not any other changes necessary to get this merged, I'd like to address this in a follow-up PR.

Reply:
Sure